src/view/shell/bottom-bar/BottomBar.tsx

```diff
@@ -47,6 +49,7 @@ export const BottomBar = observer(function BottomBarImpl({
     useNavigationTabState()
   const numUnreadNotifications = useUnreadNotifications()
   const {footerMinimalShellTransform} = useMinimalShellMode()
+  const {data: profile} = useProfileQuery({did: currentAccount!.did})
```

Comment:
Another where we need to handle it

Reply:
Updated this to match the above, will fall back to default avi